index.html

```diff
@@ -1,16 +1,27 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="philosophical_issues.html">Philosophical Issues</a></li>
+  <li><a href="clashes_between_ideas.html">Clashes between Ideas</a></li>
+
     <head>
```

Comment:
Move lines 10, 11 and 12 to the top of the page just under the <html> tag on line 3. The close the </head> tag after pulling in the css and the javascript.